styles.css

```diff
@@ -46,6 +46,8 @@ body{
     border-radius: 10px;
     font-size: 25px;
     margin-bottom: 40px;
+    margin-bottom: 40px;
```

Comment:
Good job implementing the border but i will highly suggest that you remove one margin-bottom from the element because it will lead to linters error.